test/test_unstructured_2d.jl

```diff
@@ -52,7 +52,7 @@ isdir(outdir) && rm(outdir, recursive = true)
                                 Float32(2.052861364219655)
                             ],
                             tspan=(0.0f0, 0.25f0),
-                            RealT=Float32)
+                            RealT_for_test_tolerances=Float32)
```

Comment:
```suggestion
                            RealT_for_test_tolerances=Float32,
                            RealT=Float32)
```
I think, we need to pass both here to fix tests?

Reply:
Will be fixed with https://github.com/trixi-framework/TrixiTest.jl/pull/21.